data/json/monsters/feral_humans.json

```diff
@@ -850,5 +825,224 @@
+    "death_drops": "mon_feral_sailor_death_drops",
+    "zombify_into": "mon_zombie_sailor",
+    "anger_triggers": [ "FRIEND_DIED", "FRIEND_ATTACKED", "HURT" ],
+    "flags": [ "SEES", "HEARS", "SMELLS", "WARM", "BASHES", "GROUP_BASH", "HUMAN", "CAN_OPEN_DOORS", "PATH_AVOID_DANGER_1" ]
+  },
+  {
+    "id": "mon_feral_sailor_axe",
+    "type": "MONSTER",
+    "copy-from": "mon_feral_sailor_wrench",
+    "description": "Spattered from head to toe in large swaths of dried viscera, the deep blue of this person’s tattered naval uniform has been stained the colour of rust.  Having seemingly had enough presence of mind to consider arming itself, this former crewmen carries a crash axe in its shaking hands.  The vanes on the back of their neck are visibly bulging, their eyes bloodshot, and a mass of crisscrossing blackened vanes stand out against the pale skin of their face as they regard the area around them for perceived threats.",
+    "melee_dice_sides": 6,
+    "death_drops": "mon_feral_sailor_axe_death_drops"
+  },
+  {
+    "id": "mon_feral_sailor_mop",
+    "type": "MONSTER",
+    "copy-from": "mon_feral_sailor_wrench",
+    "description": "Clutching a stout mop in their quivering hands, this crewmen was presumably assigned to basic deck sweeping in their final moments, before the Cataclysm struck and stripped them of their humanity.  Dressed in basic naval fatigues, you’d be forgiven for believing them to be another survivor.  However, their bloodshot eyes, blackened vanes, and spastic twitching tell a far different story.",
+    "melee_dice": 2,
+    "death_drops": "mon_feral_sailor_mop_death_drops"
+  },
+  {
+    "id": "mon_feral_sailor_lug_wrench",
+    "type": "MONSTER",
+    "copy-from": "mon_feral_sailor_wrench",
+    "description": "Still dressed in a grease spattered uniform, this crewmen appears to have once been a naval engineer.  With tangled hair, oil stained working attire, and a lug wrench clenched in their fist, you assume that they were either assigned to shipside maintenance or responsible for servicing onboard aircraft.  Now all that’s left of the crewmen that this person once was is a madly twitching, psychopathic monster.",
+    "melee_dice_sides": 3,
+    "death_drops": "mon_feral_sailor_lug_wrench_death_drops"
+  },
+  {
+    "id": "mon_feral_marine_bayonet",
+    "type": "MONSTER",
+    "name": { "str": "feral marine" },
+    "description": "This twitching, bloodied person is still clad in a ragged naval uniform, the insignia of the marines visible on their coat and a corresponding tattoo on their forearm.  Clutching a rifle in their quivering hands, you see them repeatedly pull the weapon’s trigger, constantly muttering incoherently and yanking back on the gun’s charging handle every time the rifle fails to fire.  Seemingly having lost the capability to comprehend that their weapon is empty, you’d feel relatively safe were it not for the gleaming bayonet still attached to the end of their firearm.",
```

Comment:
The drops for this include shotguns, which are not what the description says it's holding.